vue-frontend/src/components/home-new/BlogSection.vue

```diff
@@ -120,8 +119,19 @@ export default {
           });
 
           for (let i = 0; i < blogs.length; i++) {
-            let date = moment(String(blogs[i].pubDate)).format("MMM DD, YYYY");
-            blogs[i].pubDate = date;
+            /** date in MMM DD, YYYY format */
+            blogs[i].pubDate =
+              new Date(blogs[i].pubDate).toLocaleDateString("en-GB", {
+                month: "short",
+              }) +
```

Comment:
I think you can use "en-US" format to display date just as live site
new Date(blogs[i].pubDate).toLocaleDateString("en-US", {
                month: "short",
                day:"numeric",
              })